moped-editor/src/views/staff/StaffListViewQueryConf.js

```diff
@@ -99,19 +99,6 @@ export const StaffListViewQueryConf = {
       width: "20%",
       type: "String",
     },
-    workgroup: {
-      width: "20%",
-      searchable: true,
-      sortable: true,
-      label: "Workgroup",
-      search: {
-        label: "Search by Workgroup",
-        operator: "_ilike",
-        quoted: true,
-        envelope: "%{VALUE}%",
-      },
-      type: "String",
-    },
```

Comment:
this removes the workgroup column from the staff page. adding it back warrants a separate issue, b/c it wasn't clear to me if the graphql abstract supports reading from nested objects. we could replace the whole table with Data Grid and probl be happy about it.

Reply:
yeah lets just griddle it